LicenseHeaderManager/LicenseHeaderManager.csproj

```diff
@@ -376,16 +370,16 @@
   <PropertyGroup>
     <UseCodebase>true</UseCodebase>
   </PropertyGroup>
-  <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
-  <Import Project="$(MSBuildThisFileDirectory)..\packages\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.14.0.23-pre\tools\VSSDK\Microsoft.VsSDK.targets" />
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Import Project="$(VSToolsPath)\VSSDK\Microsoft.VsSDK.targets" Condition="'$(VSToolsPath)' != ''" />
```

Comment:
Please import the "Microsoft.VsSDK.targets" from "..\packages\Microsoft.VSSDK.BuildTools.15.5.72\tools\VSSDK\Microsoft.VsSDK.targets" in the same fashion as Line 382. We want to have our VS specific dependencies in NuGet Packages so it is easier to build the Project on a BuildServer where VS might not be installed.